lib/ae_mdw/db/mutations/contract_create_mutation.ex

```diff
@@ -40,6 +45,6 @@ defmodule AeMdw.Db.ContractCreateMutation do
     state
     |> State.inc_stat(:contracts_created)
     |> State.cache_put(:ct_create_sync_cache, contract_pk, txi)
-    |> DBContract.logs_write(nil, txi, txi, call_rec)
+    |> DBContract.logs_write(block_index, txi, txi, call_rec)
```

Comment:
Can we update the type signature for `Db.Contract.logs_write/4` not to allow `nil` as well? https://github.com/aeternity/ae_mdw/blob/master/lib/ae_mdw/db/contract.ex#L171

Reply:
done